user/docs/faq.md

```diff
@@ -0,0 +1,195 @@
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Troubleshooting & FAQ
+
+This page lists some common problems when installing or running NeoForge modpacks. If your problem isn't covered here, please see the [Getting Support][support] section.
+
+## Installing Java or NeoForge
+
+### I downloaded the installer file, but I have no way of running it!
+
+Please see the [Installing Java][installjava] section. Yes, even if you already have Minecraft: Java Edition installed.
+
+### I installed Java, but I'm still getting the wrong version!
+
+This likely means that your PATH variable is not set, or set incorrectly. The PATH variable is a system variable that tells your computer where to find Java. To fix it, do the following depending on your operating system:
+
+<Tabs defaultValue="windows">
+  <TabItem value="windows" label="Windows">
+Download and run the [Jarfix program][jarfix].
+  </TabItem>
+  <TabItem value="macos" label="MacOS">
+Open Finder. In Finder, open the Applications/Utilities folder and double-click Terminal.
+
+Run the following commands:
+
+- `echo export "JAVA_HOME=\$(/usr/libexec/java_home)" >> ~/.zshenv`
+- `echo export "PATH=$PATH:$JAVA_HOME/bin" >> ~/.zshenv`
+
+Then, close Terminal and try again.
+  </TabItem>
+  <TabItem value="linux" label="Linux">
+Many common Linux distributions have dedicated programs for managing the PATH variable. See their respective Java installation instructions:
+
+<ul>
+  <li>[Installing Java on Arch][arch]</li>
+  <li>[Installing Java on Debian][debian]</li>
+  <li>[Installing Java on Fedora][fedora]</li>
+  <li>[Installing Java on Ubuntu][ubuntu]</li>
+</ul>
+
+If you use a different distribution, you need to either look up the method for your specific distribution, or set the PATH variable manually through the console.
+
+To do so, open the terminal for your Linux distribution. Common names would be `GNOME Terminal` or `Konsole`, however it may vary depending on your exact setup.
+
+Find the location where Java is stored. Often, this will be something like `/usr/lib/jvm/java-21-openjdk-amd64`.
+
+Run the following commands:
+
+- `echo export "JAVA_HOME=/usr/lib/jvm/java-21-openjdk-amd64/" >> ~/.bashrc` (swapping out the path for the actual path if needed)
+- `echo export "PATH=$PATH:$JAVA_HOME/bin" >> ~/.bashrc`
+
+Then, close the terminal and try again.
+  </TabItem>
+</Tabs>
+
+### I am getting an error saying "Could not find or load main class @user_jvm_args.txt", what do I do?
+
+There are two common reasons for this:
+
+- You are running an outdated version of Java. Please see the [Installing Java][installjava] section.
+- You have multiple versions of Java installed on your computer, possibly installed one of them just right now, but your computer is still using the old one. This means that your PATH variable is not set correctly. Please follow the steps outlined [here][wrongjava].
+
+## Playing NeoForge
+
+### Early Display Errors
```

Comment:
I don't recall if it's possible, but if it is, I would suggest adding a comment here (that isn't visible in the rendered page) to note that the heading is used for a short link, and therefore changing it should be done carefully to avoid invalidating the link or coordinated to update the short link correspondingly.

Reply:
Simple HTML comment appears to do the trick.